### ISSUE TITLE
argdown doesn't load properly

### ISSUE THREAD

**Issue by rare18** [christianvoigt/argdown#92]
Installed vs code (newest version) plus the argdown extension and activated the argdown theme, but it doesn't load properly on 2 different windows 10 machines. The theme turns to a light theme but the two-pane layout as shown in the argdown sandbox does not show nor are the buttons for html and map visible.
Do I have to set a path somewhere or what could be the reason for this behavior? Error messages when executing commands such as export map are 'command argdown(etc) not found'. Thanks for your help,

**Comment by christianvoigt** [comment]
VS Code will only load the Argdown extension if you have opened an Argdown document. This is quite clever as it enables you to work with many different extensions without slowing the application down.

So you have to create a file with an `.argdown` or `.ad` file extension first, before you can start using the extension. Alternatively you can set the "Language Mode" to "Argdown" in the lower right corner of VS Code, using any file you like. This will also load the extension.

Does that fix your problem? We tested the extension on Windows so it should work. 

I will soon release a new version of the extension with an improved README. I simply did not think about adding this explanation to the README as VS Code (and other code editors) work like this in general.

**Comment by rare18** [comment]
Dear Christian, thanks for the quick response. Yes, that did the trick. I actually saved the new file but didn't know the extension and didn't see one reminiscent of Argdown in the safe file drop-down list so I tried one myself but it was the wrong one. I have now looked again and seen that the extension name  appears not in alphabetical order but at the end of the list. Yes, an additional sentence in the installation instructions would be helpful. Best regards and congratulations to this very interesting software. I previously used rationale but this is a different approach that actually has much to recommend it. Greetings Rainer
